scalpel-core/benchmarks/Main.hs

```diff
@@ -55,3 +64,11 @@ manySelectNodes i testData = flip scrape testData
                            $ text
                            $ foldr' (//) (tagSelector "tag")
                            $ replicate (i - 1) (tagSelector "tag")
+
+measureMemory :: T.Text -> IO ()
+measureMemory t = do
+    m <- measure (nf (scrapeStringLike t) (texts "h2")) 1
+    let pma = (show . measPeakMbAllocated . fst) m
```

Comment:
Looks like this was introduced in criterion 1.6.

Could you try adding that to extra-deps in stack.yaml? If that doesn't work let's just submit as is and I'll fiddle with the github actions to exclude benchmarks on older resolvers.

Reply:
The CI is failing on older stack resolvers that have a version of `criterion-measurement` before `measPeakMbAllocated` was introduced.

I think you can fix those by adding an `extra-dep` with a more recent version, e.g. 0.2.3.0, similar to [here](https://github.com/fimad/scalpel/commit/2608119ff248aa6d87d79a064255f1c8d21cd93b) (I don't think the sha is actually necessary).